inst/rmarkdown/templates/ieee_article/resources/template.tex

```diff
@@ -335,7 +335,7 @@
 \bibliographystyle{$if(biblio-style)$$biblio-style$$else$plainnat$endif$}
 $endif$
 $if(biblatex)$
```

Comment:
Does IEEE require using `biblatex`? If so, I feel the function `rticles::ieee_article` should have a default option `citation_package = "biblatex"`.

Reply:
IEEE officially uses bibtex [IEEE template](https://www.ieee.org/conferences/publishing/templates.html), and since biblatex its the natural successor it makes sense to use biblatex as a default